src/helpers/parseSlug.js

```diff
@@ -0,0 +1,44 @@
+var childProcess = require('child_process')
```

Comment:
Suggestion to rename this is into a general `git` helper. This could then be a standard fallback method for flag, branch, etc (unknown what etc could be)

Reply:
I'm not going to refactor in this PR, but will add a task to do so